package-examples/kustomize-pkg/README.md

```diff
@@ -0,0 +1,163 @@
+# kustomize-pkg
+
+This is a simple package that shows how kpt packages can be used instead of remote bases in kustomize.  That allows you to take advantage of kpt's rebase and local, in-place edits.
+
+## Steps
+
+1. [Fetch the package](#fetch-the-package)
+2. [View the package contents](#view-the-package-contents)
+3. [kustomize the config](#kustomize-the-config)
+4. [Render the kustomization](#render-the-kustomization)
+5. [Apply the package](#apply-the-package)
+
+### Fetch the package
+
+Get the example package on to local using `kpt pkg get`. Note that this package is for this example, wihin that package we are also using the nginx sub-package which is an alternative to having a remote base. 
+
+```shell
+$ kpt pkg get https://github.com/GoogleContainerTools/kpt.git/package-examples/kustomize-pkg
+
+fetching package /package-examples/kustomize-pkg from https://github.com/GoogleContainerTools/kpt to kustomize-pkg
+```
+
+
+### View the package contents
+
+List the package contents in a tree structure.
+
+```shell
+$ kpt pkg tree kustomize-pkg/
+
+PKG: kustomize-pkg
```

Comment:
While the output of `kpt pkg tree` here is accurate, it is kinda misleading. It hides the `bases` and `overlays` directories. It seems like we remove any directories that doesn't have any KRM resources inside them.

Reply:
it's a good observation and I think we should absolutely see if we can fix the tree output.

Reply:
you can use `tree` command for now